packages/esm-service-queues-app/src/home.component.tsx

```diff
@@ -6,7 +6,7 @@ import DefaultQueueTable from './queue-table/default-queue-table.component';
 const Home: React.FC = () => {
   return (
     <>
-      <PatientQueueHeader />
+      <PatientQueueHeader showLocationDropdown />
```

Comment:
Should this have a value?  Is the intention that this is true?  Can we please be explicit and set a value of true?